ModuleConfig.cfc

```diff
@@ -32,7 +32,9 @@ component {
 			// Enable orm auto default includes: If true and an object doesn't have any `memento` struct defined
 			// this module will create it with all properties and relationships it can find for the target entity
 			// leveraging the cborm module.
-			ormAutoIncludes = true
+			ormAutoIncludes = true,
+			// The default value for getters which return null
+			nullDefaultValue = ''
```

Comment:
Do you really think it is necessary to add this? I am pondering on it, and I really can't think of a case where I would want to change it. I think just defaulting to an empty string is more than 95% of the cases. Plus, if you want defaults on a per property, you can still use the `defaults` inside of the `this.memento` struct.

Reply:
Yes, I absolutely do.  It will not affect existing code using mementifier, but it prevents me from having to redeclare `this.memento.defaults` all over the place.

I'm also kind of opinionated on using an empty string to represent what would be a JSON object - I don't think it represents type correctly. :)   I made note of that in the issue this resolves: https://github.com/coldbox-modules/mementifier/issues/11

Reply:
Also, if true null support were enabled, I could configure this to return an actual null.

Reply:
Fair enough 🥇